components/publishers/index.jsx

```diff
@@ -0,0 +1,167 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import shortid from 'shortid';
+
+import ClassNames from 'classnames';
+
+import { PUBLISHERS } from '../../utilities/constants';
+
+import Avatar from '../avatar';
+import MediaObject from '../media-object';
+import Button from '../button';
+
+// This component's `checkProps` which issues warnings to developers about properties when in development mode (similar to React's built in development tools)
+import checkProps from './check-props';
+import componentDoc from './docs.json';
+
+const propTypes = {
+	/**
+	 * Unique ID for the component to enable ARIA functioning and keyboard interactions
+	 */
+	id: PropTypes.string,
+	/**
+	 * Function callback on click of onAddUser button
+	 */
+	onAddUser: PropTypes.func,
+	/**
+	 * Function callback on click of onAttach button
+	 */
+	onAttach: PropTypes.func,
+	/**
+	 * Function callback on change of textarea content
+	 */
+	onChange: PropTypes.func.isRequired,
+	/**
+	 * Function callback on click of submit button
+	 */
+	onSubmit: PropTypes.func.isRequired,
+	/**
+	 * Placeholder of the text area
+	 */
+	placeholder: PropTypes.string,
+	/**
+	 * This label appears above the text area
+	 */
+	label: PropTypes.string,
```

Comment:
According to the SLDS site, there is no visible label. Therefore this should be `assistiveText.label`

Reply:
<img width="760" alt="Screenshot 2019-04-27 at 4 08 11 PM" src="https://user-images.githubusercontent.com/24983958/56848444-b04bf900-6906-11e9-9a83-4c26d87cfbca.png">

I added `label` prop as there is a visible label for textarea in this example `To: My followers`. I can update the name to a better one if this is confusing.